compiler-core/src/language_server/tests/completion.rs

```diff
@@ -1144,6 +1144,77 @@ pub fn main() {
     );
 }
 
+#[test]
+fn completions_for_an_import_not_from_indirect_dependency() {
+    let code = "import gleam
+
+pub fn main() {
+  0
+}";
+    let dep = "";
+
+    let mut io = LanguageServerTestIO::new();
+    let _ = io.hex_dep_module("indirect_package", "indirect_module", "");
+    let test_project = TestProject::for_source(code).add_hex_module("example_module", dep);
+    let mut engine = test_project.build_engine(&mut io);
+
+    // Manually add an indirect dependency to the project
+    let compiler = &mut engine.compiler.project_compiler;
```

Comment:
This feels kinda bad but I couldn't think of a better way to simulate a dep of dep. I did confirm that without the actual change in `engine.rs` this test fails though. Testing out the change in IDE looks good